dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/ObjectCreationNode.java

```diff
@@ -37,25 +57,62 @@ public ObjectCreationNode(
         this.constructor = constructor;
         this.arguments = arguments;
         this.classbody = classbody;
+        this.receiver = receiver;
```

Comment:
Initialize in the same order as the parameters.

Reply:
Thanks!